README.md

```diff
@@ -26,13 +26,11 @@ So to get x, 5 * $log_2 (10000)$ / $log_2 (1000)$, which comes out to (66.435 /
 find the same element in a 10,000 tree.
 
 If it took 100 seconds it could be because...
-1. When we added 9,000 elements the tree may not still be balanced (if it ever was). The height of the subtrees could be completely different. Because the time to search the binary tree is
-   dependent on the height of the subtree it exists in, if its in a subtree that is very skewed and unbalanced (like a chain) it will take longer. My reasoning behind this is because instead of with a
-   node with two children where the number of comparisons possible is cut in half annd it grows logarithmically, a chain only has one child, so the comparisons made only decreased by one. So it would be
-   more linear. This would be a worst case scenario.
-2. Asymptotic complexity only considers the algorithm and not the machine you run it on. If the first time you run it on a computer with multiple cores, a powerful processor and CPU, and tons of storage
+1. A binary search tree is not by defintion full or balanced. The number of comparisons is dependent on the height of the subtree it exists in, if its in a subtree that is very skewed and unbalanced, more specifically a chain then the comaprisons it will need to make is however many elements are before it. My reasoning behind this is because instead of a node with two children where the number of comparisons possible is cut in half and run time grows logarithmically, a chain only has one child. So if the element ends up being at the end of a long chain, the amount of comparisons it needs to make is linear. This is the worst case of an unbalanced, binary search tree. Making sure that my argument is correct I looked up the runtime for a skewed, binary tree. https://www.geeksforgeeks.org/complexity-different-operations-binary-tree-binary-search-tree-avl-tree/
+   So doing the math behind it, $$\ 5/ log_2 (1000) = x / 10,000 = 50,000/log_2(1000) = 5000$$, so this is definetly a possibility.
```

Comment:
Not following you here. You're assuming linear runtime and getting 1000x when multiplying the input size by 10?